Sming/Libraries/ModbusMaster/samples/general/component.mk

```diff
@@ -0,0 +1,2 @@
+ARDUINO_LIBRARIES = ModbusMaster
+COMPONENT_INCDIRS = include
```

Comment:
You don't need COMPONENT_INCDIRS here, but if you want to keep it you should also add `COMPONENT_SRCDIRS = app` for consistency. (These are the defaults.)

Reply:
Also add `DISABLE_SPIFFS=1`

Reply:
Minor point, generally if possible use `:=` instead of `=` (i.e. immediate expansion rather than deferred expansion)